src/dust_afwa_mod.F90

```diff
@@ -7,9 +7,7 @@ module dust_afwa_mod
 !  06/2023, Restructure for CATChem, Jian.He@noaa.gov
 
   use catchem_constants, only : kind_chem, g=>con_g, pi=>con_pi
-  use catchem_config, only: num_chem,num_emis_dust,&
-                            p_dust_1,p_dust_2,p_dust_3,p_dust_4,p_dust_5, &
-                            p_edust1,p_edust2,p_edust3,p_edust4,p_edust5
+  use catchem_config, only: num_chem,num_emis_dust,p_dust_1, p_edust1
```

Comment:
This may be related to another issue. I'm thinking of undefining num_chem in catchem_config.  num_chem would be different for different configurations. Maybe we can assign ntchm to num_chem?

Reply:
Lets handle this in a future PR